pyimzml/ImzMLWriter.py

```diff
@@ -105,16 +106,25 @@
 """
 
 class ImzMLWriter(object):
-    def __init__(self, output_filename, mz_dtype=np.float64, intensity_dtype=np.float32):
+    def __init__(self, output_filename, mz_dtype=np.float64, intensity_dtype=np.float32, mz_compression=False, intensity_compression=False):
+        '''"output_filename" is used to make the base name by removing the extension (if any).
+        two files will be made by adding ".ibd" and ".imzML" to the base name
+        processed or continuous mode will be chosen automatically. Identical mz lists will only be written once.
+        Force process mode by setting "instance.hashes = None"
+        "compression" must be True or False or indicate a number to round to.
```

Comment:
I agree that mz and intensities should be treated separately, I was thinking towards 
```python
def __init__(..., mz_compression=NoCompression(),
                  intensity_compression=ZlibCompression(...))
```